tools/azure-sdk-tools/devtools_testutils/aio/proxy_testcase_async.py

```diff
@@ -0,0 +1,54 @@
+from ..proxy_testcase import (
+    get_test_id,
+    start_record_or_playback,
+    transform_request,
+    stop_record_or_playback,
+)
+
+from azure.core.pipeline.transport import AioHttpTransport
+
+from azure_devtools.scenario_tests.utilities import trim_kwargs_from_test_function
+
+def RecordedByProxyAsync(func):
+    async def record_wrap(*args, **kwargs):
+        test_id = get_test_id()
+        recording_id = start_record_or_playback(test_id)
+
+        def transform_args(*args, **kwargs):
+            copied_positional_args = list(args)
+            request = copied_positional_args[1]
+
+            # TODO, get the test-proxy server a real SSL certificate. The issue here is that SSL Certificates are
+            # normally associated with a domain name. Need to talk to the //SSLAdmin folks (or someone else) and get
+            # a recommendation for how to get a valid SSL Cert for localhost
+            kwargs["connection_verify"] = False
```

Comment:
As discussed. I'll help figure out the trust methodology for the certificate. We should remove this before we merge.